addons/sale_product_configurator/static/src/js/product_configurator_controller.js

```diff
@@ -115,12 +115,13 @@ var ProductConfiguratorFormController = FormController.extend({
         var initialProduct = this.initialState.data.product_template_id;
         var changed = initialProduct && initialProduct.data.id !== productTemplateId;
         var data = this.renderer.state.data;
+        var quantity = initialProduct.context && initialProduct.context.default_quantity ? initialProduct.context.default_quantity : data.quantity;
```

Comment:
Doesn't this mean that the existing qty will be always kept even if the qty is changed in the wizard ? :thinking: